src/elements/content-uploader/ItemList.js

```diff
@@ -16,18 +16,27 @@ import '@box/react-virtualized/styles.css';
 import './ItemList.scss';
 
 type Props = {
+    freemiumUpsellCTACb?: ?() => void,
```

Comment:
Does the Cb stand for callback? is there convention to use that or can we rename it to `onFreemiumUpsellCTAClick`?